src/GrumPHP/Task/PhpCsFixer.php

```diff
@@ -75,44 +83,68 @@ public function run(ContextInterface $context)
         $arguments->addOptionalCommaSeparatedArgument('--fixers=%s', $config['fixers']);
         $arguments->add('fix');
 
+        if ($context instanceof RunContext && $config['config'] !== null) {
```

Comment:
Woops, overlooked that one :)
It's fixed on master: https://github.com/phpro/grumphp/commit/bd8413ec6a9baf219f5625ff9b9c0dd5986b4136